File: src/primus/pawel/Exercise_122.java
package primus.pawel;

// Small enough? - Beginner
// 7kyu
// https://www.codewars.com/kata/57cc981a58da9e302a000214/train/java
// 2021-05-11 T:23:05:10
class Exercise_122 {

    public static boolean smallEnough(int[] a, int limit) {
        for (int i = 0; i < a.length; i++) {
            if (a[i] > limit) {
                return false;
            }
        }
        return true;
    }

}

File: src/JS/Exercise_122.js

// Leap Years
// 7 kyu
// https://www.codewars.com/kata/526c7363236867513f0005ca/train/javascript
// 2022-06-13 T:07:40:35

function isLeapYear(year) {
    if (year % 4 !== 0) {
        return false
    }
    return !(year % 100 === 0 && year % 400 !== 0);
}